data/lm/generate_package.py

```diff
@@ -112,23 +112,27 @@ def main():
         required=True,
         help="Path of vocabulary file. Must contain words separated by whitespace.",
     )
-    parser.add_argument("--package", required=True, help="Path to save scorer package.")
+    parser.add_argument(
+        "--package",
+        required=True,
+        help="Path to save scorer package."
+    )
     parser.add_argument(
         "--default_alpha",
         type=float,
-        required=True,
+        default=0.75,
         help="Default value of alpha hyperparameter.",
     )
     parser.add_argument(
         "--default_beta",
         type=float,
-        required=True,
+        default=1.85,
         help="Default value of beta hyperparameter.",
     )
     parser.add_argument(
         "--force_utf8",
         default="",
-        help="Boolean flag, force set or unset UTF-8 mode in the scorer package. If not set, infers from the vocabulary.",
+        help="Boolean flag, force set or unset UTF-8 mode in the scorer package. If not set, infers from the vocabulary. Using this wrong can result in a 'Segmentation fault' error at the model evaluation.",
```

Comment:
Using the wrong scorer can lead to inconsistent behavior but not segmentation fault. If you've seen that happen, please create an issue with steps to reproduce.

Reply:
Created an issue: https://github.com/mozilla/DeepSpeech/issues/2875